v6/remove-old-lokalise-prs.js

```
@@ -0,0 +1,52 @@
+
+  //Fetch PRs
+  octo.rest.pulls.list({
+    owner: owner,
+    repo: repo,
+  }).then(prs => {
+    //Filter PRs
+    let filteredPrs = prs.data.filter(pr => {
+      let regex = new RegExp('^Lokalise:[ _a-zA-Z0-9]+');
+      return regex.test(pr.title) && pr.number != currentPR;
+    });
+    if (filteredPrs && filteredPrs.length) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)